### PR TITLE
remove old commented-out code

### PR DESCRIPTION
**Pull request by rodolforg**

### CODE REVIEW

**Comment by rodolforg** [comment]
since 2005

**Comment by morevnaproject** [comment]
I think it's okay to remove that ^__^

File: synfig-studio/src/gui/trees/layertree.cpp [hunk 1]
```diff
@@ -152,33 +140,8 @@ LayerTree::LayerTree():
 	attach(*layer_amount_hscale, 1, 2, 1, 2, Gtk::EXPAND|Gtk::FILL, Gtk::SHRINK, 1, 1);
 	layer_amount_adjustment_->signal_value_changed().connect(sigc::mem_fun(*this, &studio::LayerTree::on_amount_value_changed));
 
-	Gtk::Image *icon;
 	Gtk::IconSize iconsize(Gtk::ICON_SIZE_SMALL_TOOLBAR);
 
-	SMALL_BUTTON(button_raise,"gtk-go-up","Raise");
```
Comment (ice0) [inline synfig-studio/src/gui/trees/layertree.cpp:158]: It seems this code has not been commented out, but I can’t figure out where these buttons were?

Comment (rodolforg) [reply synfig-studio/src/gui/trees/layertree.cpp:158]: They're not used.
The buttons are in the correspondent dock class

Comment (ice0) [reply synfig-studio/src/gui/trees/layertree.cpp:158]: Ok

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**